.github/FUNDING.yml

```diff
@@ -1,2 +1,3 @@
+github: betonquest
 open_collective: betonquest
-custom: [ "https://mcmodels.net/?wpam_id=3" ]
+custom: [ "https://mcmodels.net/?wpam_id=3", "mcmodels.net Affiliate Link" ]
```

Comment:
It seems like this change broke the mcmodels link.

The [documentation](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/displaying-a-sponsor-button-in-your-repository#about-funding-files) is a bit unclear on how custom links work.

Reply:
The new PR fixes this change, it is indeed invalid.